### PR TITLE
release: jdk17.0.6+10

### PR DESCRIPTION
**Pull request by steelhead31**
shipped:
x64 linux
aarch64 linux
ppc64le
arm32
s390x
alpine

### CODE REVIEW

File: linux/jdk/debian/src/main/packaging/temurin/17/debian/changelog [hunk 1]
```diff
@@ -1,3 +1,9 @@
+temurin-17-jdk (17.0.6.0.0+10) STABLE; urgency=medium
+
+  * Eclipse Temurin 17.0.6.0.0+10 release.
+
+ -- Eclipse Adoptium Package Maintainers <temurin-dev@eclipse.org>  Wed, 18 Jan 2023 09:43:20 +0000
```
Comment (zdtsw) [inline linux/jdk/debian/src/main/packaging/temurin/17/debian/changelog:5]: think we usually use the date when the installer should be built not the one GA tag ready

**Review by zdtsw — APPROVED** [review]

File: linux/jdk/debian/src/main/packaging/temurin/17/debian/changelog (hunk 1 above)
Comment (gdams) [reply linux/jdk/debian/src/main/packaging/temurin/17/debian/changelog:5]: this date looks good to me, the GA date makes more sense as it allows users to compare against the tags

**Review by gdams — APPROVED** [review]

**Comment by gdams** [comment]
Kicked off the Linux installer pipeline https://ci.adoptopenjdk.net/job/adoptium-packages-linux-pipeline_new/89/

**Pull request merged**